### PR TITLE
[RU] Translation update:

### PR DESCRIPTION
**Pull request by MasterGroosha**
• Added translation to Welcome timeline
• Doorkeeper: changed date format to more common in Russia
• Doorkeeper: added translation to scopes
• Added translation to Dashboard settings (I didn't have a chance to test it on Live server, so some strings may be a bit incorrect)
• Added translation to Relays settings
• Added translation to Filters settings
• Some other small fixes and improvements

### CODE REVIEW

**Comment by MasterGroosha** [comment]
I guess I have to normalize it somehow, but I'm currently not able to do that :(

File: app/javascript/mastodon/locales/ru.json [hunk 1]
```diff
@@ -84,8 +84,8 @@
   "confirmations.domain_block.message": "Вы на самом деле уверены, что хотите блокировать весь {domain}? В большинстве случаев нескольких отдельных блокировок или глушений достаточно.",
   "confirmations.mute.confirm": "Заглушить",
   "confirmations.mute.message": "Вы уверены, что хотите заглушить {name}?",
-  "confirmations.redraft.confirm": "Delete & redraft",
-  "confirmations.redraft.message": "Are you sure you want to delete this status and re-draft it? You will lose all replies, boosts and favourites to it.",
+  "confirmations.redraft.confirm": "Удалить и повторить",
```
Comment (Gargron) [inline app/javascript/mastodon/locales/ru.json]: Povtorit is not quite right here I feel... More like udalit i pravit?

**Review by Gargron — APPROVED** [review]

File: app/javascript/mastodon/locales/ru.json (hunk 1 above)
Comment (MasterGroosha) [reply app/javascript/mastodon/locales/ru.json]: @Gargron I think you're right. Changed it to "исправить".

**Comment by Gargron** [comment]
Sent a PR with normalize against your master. The workflow for adding commits to PRs that are based on fork branches is really complicated...

**Comment by MasterGroosha** [comment]
@Gargron, I'm really sorry, will try to do normalization myself next time. Thank you for that PR!

File: app/javascript/mastodon/locales/ru.json [hunk 2]
```diff
@@ -194,7 +194,7 @@
   "notifications.column_settings.reblog": "Продвижения:",
   "notifications.column_settings.show": "Показывать в колонке",
   "notifications.column_settings.sound": "Проигрывать звук",
-  "notifications.group": "{count} notifications",
+  "notifications.group": "{count} уведомл.",
```
Comment (MasterGroosha) [inline app/javascript/mastodon/locales/ru.json]: is it possible to use correct plurals here? Something like "trends.count_by_accounts" key (`"trends.count_by_accounts": "{count} {rawCount, plural, one {person} other {people}} talking"`)

**Pull request merged**